content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
+    EOF
+    docker-compose up
+   {{% /tab %}}
+   {{% tab "docker" %}}
+    docker run -p 4222:4222 -ti nats:latest
+   {{% /tab %}}
+   {{% tab "nats-server binary" %}}
+    nats-server -a 0.0.0.0 -p 4222
+   {{% /tab %}}
+ {{% /tabs %}}
 
-# add css plugin
-[[params.plugins.css]]
-link = "define plugins path"
+### Running your first wasmCloud Host
+In a separate terminal window, run the following command to launch an interactive wasmCloud REPL environment, including a preconfigured wasmCloud host.
+```shell
+wash up
+```
+At any time, you can type `help` to see a printed list of subcommands in the log output window in the bottom-right pane. To exit the repl, you can type `exit`, `logout`, `q` or `:q!`.
+![REPL](../wash_up.png)
 
-# add js plugin
-[[params.plugins.js]]
-link = "define plugins path"
+To see a list of running hosts, run the following command in the REPL:
+```shell
+wash> ctl get hosts
+```
+You should see one host running, and you can view its inventory by running the following command with the **Host ID** found in the Output window:
+```shell
+wash> ctl get inventory NXYZABC...
+```
+You'll see output similar to the following (public key output truncated for documentation):
+```shell
+               Host Inventory (NBMRMKELSYA645ZBGN6AEYJU6GV...)                
+                                                                                                        
+hostcore.os                                                      macos                                        
+hostcore.osfamily                                                unix                                         
+hostcore.arch                                                    x86_64                                       
+repl_mode                                                        true                                         
+                          No actors found                                             
+                                                                                                        
+Provider ID               Link Name                  Image Reference   
+VDHPKGFKDI34Y4RN4PWWZH... default                    N/A               
+VAHNM37G4ARHZ3CYHB3L34... __wasmcloud_lattice_cache  N/A               
+```
+Currently on this Host, we have a few labels that show the environment this Host is running on, and two default providers. The first, starting with `VDHPK`, is the `extras` provider, which provides an interface for wasmCloud resources to generate random numbers, GUIDS, etc. The second, starting with `VAHNM`, is the lattice cache provider, which provides a distributed cache to all hosts running in this lattice. We'll talk more about what a **Lattice** is in the [Platform Builder](../../platform-builder) section of the documentation.
 
+#### Running an Actor
+We can start scheduling Actors and Providers right away on this Host using the `ctl start` command. To begin, we'll start our `Echo` sample actor.

```

Comment:
I've gone back and forth on this, and editorial style and copy editing say that actors and providers shouldn't be capitalized.

Reply:
Host here is capitalized and doesn't need to be.

Reply:
re: this and above `Host` comment, I'll convert the Host, Actor, and Provider mentions to be lowercased.